test/e2e/features/pageObjects/hearingRecordingsCase.js

```diff
@@ -1,5 +1,7 @@
 
 var BrowserWaits = require("../../support/customWaits");
+
+var BrowserUtil = require('../../../ngIntegration/util/browserUtil');
```

Comment:
very minor - happy to approve without, but could `const` be used instead of `var` for file consistency?